inference-engine/src/mkldnn_plugin/nodes/mkldnn_adaptive_pooling.h

```diff
@@ -0,0 +1,44 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <ie_common.h>
```

Comment:
Please check, do you really need this header

Reply:
Done